pipeline_steps/common.groovy

```diff
@@ -3,7 +3,8 @@ import groovy.json.JsonOutput
 
 // Install ansible on a jenkins slave
 def install_ansible(){
-  sh """#!/bin/bash
+  sh """#!/bin/bash -xe
+    cd ${env.WORKSPACE}
```

Comment:
Actually I see your point. The horizon and kibana test functions do creation distinct .venv venvs. :-/